### ISSUE TITLE
[fix] pfe-autocomplete | Fix code debugging and missing disconnectedCallback

### ISSUE THREAD

**Issue by Homa** [patternfly/patternfly-elements#326]
1- remove debugger from this line
https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-autocomplete/src/pfe-autocomplete.js#L291

2- add super.disconnectedCallback(); in these two places:
- https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-autocomplete/src/pfe-autocomplete.js#L97
- https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-autocomplete/src/pfe-autocomplete.js#L414

**Comment by castastrophe** [comment]
@Homa This is not something that should be assigned to me.  I can pull it into a sprint if it is prioritized but several others on the team, yourself included, might get time before I'm able to.  I also added the component name into the title of the issue and the `bug` label to add clarity.  Finally, I've assigned this to the Components backlog since it's work that needs to be prioritized against the other component issues.  Thanks!
